### PR TITLE
[WIP] Feature/improve prometheus stack

### PR DESCRIPTION
**Pull request by AlexSonar**
The state creates an overridden cluster but does not deploy Prometheus yet

### CODE REVIEW

File: examples/argocd-with-applications/apps/argocd.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+"apiVersion": "argoproj.io/v1alpha1"
```
Comment (akastav) [inline examples/argocd-with-applications/apps/argocd.yaml:1]: Please remove the apps folder and sensitive data

File: examples/argocd-with-applications/main.tf [hunk 2]
```diff
@@ -35,13 +35,16 @@ module "network" {
 
 module "kubernetes" {
   depends_on = [module.network]
-  source     = "github.com/provectus/sak-kubernetes"
+  # source     = "github.com/provectus/sak-kubernetes"
+  # source     = "../../../sak-kubernetes-temporrary"
+  source     = "github.com/AlexSonar/sak-kubernetes"
```
Comment (akastav) [inline examples/argocd-with-applications/main.tf:40]: Could you change this back?

File: examples/argocd-with-applications/main.tf [hunk 3]
```diff
@@ -118,25 +121,27 @@ module "nginx-ingress" {
...
+#   config_path       = "${path.module}/kubeconfig_${var.cluster_name}"
+#   certificates_arns = [module.clusterwide.this_acm_certificate_arn]
+#   cluster_oidc_url  = module.kubernetes.cluster_oidc_url
 # }
 
+module "prometheus" {
+  depends_on   = [module.argocd]
+  # source       = "github.com/provectus/sak-prometheus"
+  # source       = "../../../prometheus/sak-prometheus"
+  source       = "github.com/AlexSonar/sak-prometheus"
```
Comment (akastav) [inline examples/argocd-with-applications/main.tf:139]: The same things, please don't change source in PR

File: examples/argocd-with-applications/variables.tf [hunk 4]
```diff
@@ -1,23 +1,26 @@
 variable "cluster_name" {
-  default     = "swiss-army"
+  # default     = "swiss-army"
+  default     = "swiss-army-szotov"
```
Comment (akastav) [inline examples/argocd-with-applications/variables.tf:3]: Why do you change default values? You can override it in main.tf

**Review by akastav — CHANGES_REQUESTED** [review]

File: examples/argocd-with-applications/apps/argocd.yaml (hunk 1 above)
Comment (AlexSonar) [reply examples/argocd-with-applications/apps/argocd.yaml:1]: appsargoproj
common
hydrosphere

File: examples/argocd-with-applications/main.tf (hunk 2 above)
Comment (AlexSonar) [reply examples/argocd-with-applications/main.tf:40]: I created forks of repositories [ "sak-kubernetes",  "sak-prometheus", ] temporarily,

**Pull request closed without merging**